### PR TITLE
fix(publish): update `publish.ts` to sern 1.1.0-beta

### PR DESCRIPTION
**Pull request by jacoobes**
This had some outdated code only available to 1.0.0-beta. Also added a version of sern on top of the code

### CODE REVIEW

**Comment by jacoobes** [comment]
@EvolutionX-10 MERGEEE

File: TypeScript/publish.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
+// sern-handler 1.1.0-beta
```
Comment (EvolutionX-10) [inline TypeScript/publish.ts]: ts nocheck line should be at the top right? doesn't ts throw errors if not at top of the file?

Comment (EvolutionX-10) [reply TypeScript/publish.ts]: also won't it be a pain to update versions manually everytime in plugins?

**Review by EvolutionX-10 — COMMENTED** [review]
WTF I THOUGHT I REVIEWED IT DAMN IT GITHUB

File: TypeScript/publish.ts (hunk 1 above)
Comment (jacoobes) [reply TypeScript/publish.ts]: yeah true, it would. I feel like it would be nice to organize
Your call
yeah add ts-nocheck to the top

Comment (EvolutionX-10) [reply TypeScript/publish.ts]: I tested it, it actually works as long as things above it is comments

**Review by EvolutionX-10 — APPROVED** [review]
cheers

**Pull request merged**